index.js

```diff
@@ -13,4 +17,174 @@ closeHamburger.addEventListener('click', () => {
 
 hamburgerList.forEach((list) => list.addEventListener('click', () => {
   sideNav.classList.remove('visible');
-}));
\ No newline at end of file
+}));
+
+const projectList = [
+  {
+    id: '1',
+    name: 'Keeping track of hundreds of components',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release",
+    featuredImage: './assets/Snapshoot-Portfolio.png',
+    desktopImage: './assets/Snapshoot-Portfolio-desktop.png',
+    closeIcon: './assets/Enabled.png',
+    technologies: {
+      ruby: 'Ruby on rails',
+      css: 'css',
+      javascript: 'Javascript',
+      codekit: 'Codekit',
+      github: 'GitHub',
+      bootstrap: 'Bootstrap',
+      terminal: 'Terminal',
+      codepen: 'Codepen',
+    },
+    liveVersion: 'https://fayob.github.io/portfolio-setup-project/',
+    source: 'github.com/fayob/portfolio-setup-project',
```

Comment:
There is nowhere to write my comment under the README file review. About the Author, I think I am the major author of this project because each and everyone is writing his/her portfolio project and it contains our individual information.

Reply:
That's why you need to add him to the author section of your README  because he has contributed to this project